package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-polymorph",
   "description": "React components with highly customizable logic, markup and styles.",
-  "version": "1.0.1",
+  "version": "1.0.1-next.3",
```

Comment:
once this pre-release has been tested, please publish a new version to NPM, and update the version to no longer point to next, but `1.0.2`